draft-ietf-httpbis-digest-headers.md

```diff
@@ -278,6 +278,8 @@ A recipient MAY ignore any or all of the representation-data-digests in a Digest
 field. This allows the recipient to choose which digest-algorithm(s) to use for
 validation instead of verifying every received representation-data-digest.
 
+A recipient MUST ignore a representation-data-digest containing parameters.
```

Comment:
```suggestion
```

Removed this line. I'll open another issue for providing guidance in the parameter transition.